File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/PromoteFromAlphaToBetaTask.java
package com.dipien.google.play.api.task;

import com.dipien.google.play.api.App;
import com.dipien.google.play.api.GooglePlayPublisher;

public class PromoteFromAlphaToBetaTask extends BaseTask {
	
	public PromoteFromAlphaToBetaTask() {
		setDescription("Promote a current alpha to beta");
	}

	@Override
	protected void onExecute(App app) {
		GooglePlayPublisher.promoteFromAlphaToBeta(app);
	}

}

File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/PromoteFromAlphaToBetaTask.kt
package com.dipien.google.play.api.task

import com.dipien.google.play.api.App
import com.dipien.google.play.api.PublishingService

class PromoteFromAlphaToBetaTask : BaseTask() {
    override fun onExecute(app: App) {
        PublishingService().promoteFromAlphaToBeta(app)
    }

    init {
        description = "Promote a current alpha to beta"
    }
}
